machine_configuration/machine-configs-configure.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 You can use the tasks in this section to create `MachineConfig` objects that modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with machine configs, see content related to link:https://access.redhat.com/solutions/3868301[updating] SSH authorized keys, xref:../security/container_security/security-container-signature.adoc#security-container-signature[verifying image signatures], link:https://access.redhat.com/solutions/4727321[enabling SCTP], and link:https://access.redhat.com/solutions/5170251[configuring iSCSI initiatornames] for {product-title}.
 
-{product-title} supports link:https://coreos.github.io/ignition/configuration-v3_2/[Ignition specification version 3.2]. All new machine configs you create going forward should be based on Ignition specification version 3.2. If you are upgrading your {product-title} cluster, any existing Ignition specification version 2.x machine configs will be translated automatically to specification version 3.2.
+{product-title} supports link:https://coreos.github.io/ignition/configuration-v3_4/[Ignition specification version 3.4]. You should base all new machine configs you create going forward on Ignition specification version 3.4. If you are upgrading your {product-title} cluster, any existing machine configs with a previous Ignition specification will be translated automatically to specification version 3.4.
```

Comment:
"should" or "must"?

Reply:
@maxwelldb The text previously said _should_. Probably best to leave it that way. I just wanted to tighten the sentence and get rid of the passive.